src/app/utilities/api-clients/teams.js

```diff
@@ -2,9 +2,19 @@ import http from "../http";
 
 export default class teams {
     static getAll() {
-        return http.get(`/zebedee/teams`).then(response => {
-            return response.teams;
-        });
+        let config = {};
+        if (window.getEnv != null) {
+            config = window.getEnv();
+        }
+        if (config.enableNewSignIn) {
```

Comment:
can we start unifying things here? there is api to fetch new groups already created:  https://github.com/ONSdigital/florence/pull/681/files#diff-286ad46cc58e9b11c299bde801fc73666c057e8e68ff57350d8e0691597b1c91R62